hardware.inc

```diff
@@ -769,33 +778,33 @@ def rRTCREG equ $A000
 ; Screen-related constants
 ;******************************************************************************
 
-def SCRN_X   equ 160 ; width of screen in pixels
-def SCRN_Y   equ 144 ; height of screen in pixels
-def SCRN_X_B equ  20 ; width of screen in bytes
-def SCRN_Y_B equ  18 ; height of screen in bytes
-def SCRN_B   equ SCRN_X_B * SCRN_Y_B ; size of screen in bytes
-
-def SCRN_VX   equ 256 ; width of tilemap in pixels
-def SCRN_VY   equ 256 ; height of tilemap in pixels
-def SCRN_VX_B equ  32 ; width of tilemap in bytes
-def SCRN_VY_B equ  32 ; height of tilemap in bytes
-def SCRN_V_B  equ SCRN_VX_B * SCRN_VY_B ; size of tilemap in bytes
-
-def TILE_X equ  8 ; width of tile in pixels
-def TILE_Y equ  8 ; height of tile in pixels
-def TILE_B equ 16 ; size of tile in bytes (2 bits/pixel)
-
-def COLOR_B    equ 2 ; size of color in bytes (little-endian BGR555)
-    def COLORF_GREEN_LOW  equ %111_00000  ; for the low byte
-    def COLORF_RED        equ %000_11111  ; for the low byte
-    def COLORF_BLUE       equ %0_11111_00 ; for the high byte
-    def COLORF_GREEN_HIGH equ %000000_11  ; for the high byte
+def SCREEN_WIDTH_PX  equ 160 ; width of screen in pixels
+def SCREEN_HEIGHT_PX equ 144 ; height of screen in pixels
+def SCREEN_WIDTH     equ  20 ; width of screen in bytes
+def SCREEN_HEIGHT    equ  18 ; height of screen in bytes
+def SCREEN_AREA      equ SCREEN_WIDTH * SCREEN_HEIGHT ; size of screen in bytes
+
+def TILEMAP_WIDTH_PX  equ 256 ; width of tilemap in pixels
+def TILEMAP_HEIGHT_PX equ 256 ; height of tilemap in pixels
+def TILEMAP_WIDTH     equ  32 ; width of tilemap in bytes
+def TILEMAP_HEIGHT    equ  32 ; height of tilemap in bytes
+def TILEMAP_AREA      equ TILEMAP_WIDTH * TILEMAP_HEIGHT ; size of tilemap in bytes
+
+def TILE_WIDTH  equ  8 ; width of tile in pixels
+def TILE_HEIGHT equ  8 ; height of tile in pixels
+def TILE_SIZE   equ 16 ; size of tile in bytes (2 bits/pixel)
+
+def COLOR_SIZE equ 2 ; size of color in bytes (little-endian BGR555)
+    def COLOR_GREEN_LOW  equ %111_00000  ; for the low byte
+    def COLOR_RED        equ %000_11111  ; for the low byte
+    def COLOR_BLUE       equ %0_11111_00 ; for the high byte
+    def COLOR_GREEN_HIGH equ %000000_11  ; for the high byte
 def PAL_COLORS equ 4 ; colors per palette
-def PAL_B      equ COLOR_B * PAL_COLORS ; size of palette in bytes
+def PAL_SIZE   equ COLOR_SIZE * PAL_COLORS ; size of palette in bytes
 
 ; Tilemaps the BG or Window can read from (controlled by LCDC)
-def _SCRN0 equ $9800 ; $9800-$9BFF
-def _SCRN1 equ $9C00 ; $9C00-$9FFF
+def TILEMAP0 equ $9800 ; $9800-$9BFF
+def TILEMAP1 equ $9C00 ; $9C00-$9FFF
 
```

Comment:
These changes are all great. I thought it was somewhat confusing before to use X and Y to refer to _dimension_ and it was unclear why we have an underscore in front of `_SCRN0`? TILEMAP makes it clearer that it is actually the tilemap and not something to do with like the screen itself.